lib/node_modules/@stdlib/blas/base/wasm/sdsdot/test/test.main.js

```diff
@@ -0,0 +1,229 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var Float32Array = require( '@stdlib/array/float32' );
+var sdsdot = require( './../lib' );
+
+
+// TESTS //
+
+tape( 'main export is an object', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof sdsdot, 'object', 'main export is an object' );
+	t.end();
+});
+
+tape( 'the `main` method has an arity of 6', function test( t ) {
+	t.strictEqual( sdsdot.main.length, 6, 'returns expected value' );
+	t.end();
+});
+
+tape( 'the `main` method computes the dot product of `x` and `y` with extended accumulation and result', function test( t ) {
```

Comment:
```suggestion
tape( 'the `main` method computes the dot product of `x` and `y` with extended accumulation', function test( t ) {
```

Reply:
`sdsdot` -> sds -> single-precision in, double-precision accumulate, single-precision out. The result is not in extended precision.